src/plugins/graphql.js

```diff
@@ -48,22 +48,29 @@ function createWrapParse (tracer, config) {
     return function parseWithTrace (source) {
       const span = startSpan(tracer, config, 'parse')
 
-      analyticsSampler.sample(span, config.analytics)
+      let operationValue = ''
```

Comment:
nit: better practice to define and use locals in a single place.  You can initialize w/ default value (line 58) and immediately set operation name instead of having the logic spread out.